aiohttp/multipart.py

```diff
@@ -907,16 +901,14 @@ def write(self, chunk):
                     return
 
         if self._encoding == 'base64':
-            self._encoding_buffer.extend(chunk)
-
-            if self._encoding_buffer:
-                buffer = self._encoding_buffer
-                div, mod = divmod(len(buffer), 3)
-                enc_chunk, self._encoding_buffer = (
-                    buffer[:div * 3], buffer[div * 3:])
-                if enc_chunk:
-                    enc_chunk = base64.b64encode(enc_chunk)
-                    yield from self._writer.write(enc_chunk)
+            if chunk:
+                enc_chunk = base64.encodebytes(chunk)
```

Comment:
@socketpair 
I wouldn't be so strict here. When you open PR, your remote branch becomes liked with PR. Any changes there automagically happens here. Do you suggest to not do any push after that until you done requested changes? That's not cool.

Much better when contributor pings you once requested changes becomes satisfied. He can continue work on in his branch while you don't pay attention to WIP PRs.